SPECS/grub2-rpm-macros/macros.grub2

```diff
@@ -0,0 +1,20 @@
+# RPM macros for packages updating boot behavior in Mariner.
+# grub2-mkconfig should be used to regenerate /boot/grub2/grub.cfg
+#   iff: grub2-mkconfig is installed && the existing /boot/grub2/grub.cfg 
+#     seems to have been generated by mkconfig previously. 
+
+%grub2_post() \
+if [ -f /usr/sbin/grub2-mkconfig ] && grep -q -e "automatically generated by grub2-mkconfig" /boot/grub2/grub.cfg; then \
+    if grep -q "automatically generated by grub2-mkconfig" /boot/grub2/grub.cfg; then \
+        /usr/sbin/grub2-mkconfig > /boot/grub2/grub.cfg || : \
+    fi \
+fi \
+%{nil}
+
+%grub2_postun() \
```

Comment:
These macros are identical right now.. I think we should distinguish between them still (if we need to support a difference in behavior later).

Reply:
sounds good, though above comments apply here too